client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -408,22 +428,44 @@ export default React.createClass( {
 			);
 		}
 
-		const fullContent = (
+		let fullContent, compactContent, compactNoticeText;
+
+		if ( severity === 'is-error' ) {
+			fullContent = (
+					<span>
+					{ this.translate( 'Your domain {{strong}}%(domain)s{{/strong}} may be suspended because your email address is not verified.' ) }
```

Comment:
I think your missing the `domain` and `strong` arguments for your translation here